### ISSUE TITLE
Proposed Analysis: Test consensus CNV calls with GATK CNV instead of MantaSV

### ISSUE THREAD

**Issue by jharenza** [PediatricOpenTargets/ticket-tracker#356]
#### What are the scientific goals of the analysis?
To determine whether GATK CNV can be a suitable replacement for MantaSV, we need to understand how this algorithm performs in the consensus calls module. In addition, we should test whether adding GATK CNV as a 4th caller helps improve true positive calls. As of v11, we only have ~60% of samples with CNV calls in our resulting consensus files. See [ticket](https://github.com/PediatricOpenTargets/ticket-tracker/issues/354) for discussion.

#### What methods do you plan to use to accomplish the scientific goals?
1. Modify [copy_number_consensus_call](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/copy_number_consensus_call) to replace MantaSV with GATK CNV and run PBTA and TARGET NBL samples.
2. Modify [copy_number_consensus_call](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/copy_number_consensus_call) to add GATK CNV as a 4th caller and identify consensus using 2/4. Run with PBTA and TARGET NBL samples.

It may be easier to do 2 above and create an option for input of any combination of the 4.

#### What input data are required for this analysis?
```
cnv-cnvkit.seg.gz
cnv-controlfreec.tsv.gz
sv-manta.tsv.gz
cnv-gatk.seg.gz (?)
```

#### How long do you expect is needed to complete the analysis? Will it be a multi-step analysis?
1-2 weeks

#### Who will complete the analysis (please add a GitHub handle here if relevant)?
This is currently blocked by #357, but @migbro do you think updating this module's codebase is something you or @dmiller15 can tackle?

#### What relevant scientific literature relates to this analysis?

cc @afarrel @yuankunzhu

**Comment by migbro** [comment]
@sickler-alex has started working on the first part of this, which is to enable to pbta module to take in GATK input. Afterwards, analysts can take over and compare copy number calls

**Comment by sickler-alex** [comment]
I decided to open the PR for review. This is the link https://github.com/PediatricOpenTargets/ticket-tracker/issues/356

**Comment by chinwallaa** [comment]
Waiting on some methods for manuscript @zhangb1

**Comment by chinwallaa** [comment]
@ewafula to review and merge, will write a separate ticket for capturing documentation
